### PR TITLE
[Doc] Add python kernel sample data guide.

### PR DESCRIPTION
**Pull request by yoyo-wu98**
Since we have added the Python Kernel of the open-digger and there is an issue #968 asking for it, the guide of the Sample Data Usage with Python Kernel should be added.

### CODE REVIEW

**Comment by frank-zsy** [comment]
My concern is that right now we add kernel related docs into `sample_data` readme, but actually the sample data doc should not be mixed with kernel doc at all because they are totally different thing here.

Can we split the kernel related docs into another place and only reference in the sample data doc?

**Comment by zhicheng-ning** [comment]
A confusing situation occur.
In my sight：
b7261eada60a6f3bcfd1229750e3c92
In other people's insight：
67f98ef52a8ccced4386b1a4246b6fa

**Review by zhicheng-ning — CHANGES_REQUESTED** [review]

File: sample_data/README.md [hunk 1]
```diff
@@ -74,6 +76,53 @@ Start your ClickHouse container, which should be set up in the last step. Now:
...
+
+   > If you are using **Windows CMD**, all the `$(pwd)` here should be replaced by `%cd%`. And if you are using **Windows Powershell**,  all the `$(pwd)` here should be replaced by `${pwd}`.
+   >
+   > **Notice:** Pathnames of directories like "pwd" may use `\` to join the directory in some versions of Windows. We recommend using absolute paths.
+
+5. Then we can use `docker run -it --name python_notebook_name --rm -p 8888:8888 -v $(pwd):/python_kernel/notebook opendigger-jupyter-python:1.0` to create and run the container.
+
+6. Open the link in console log like `http://127.0.0.1:8888/lab?token=xxxxx`.
+
+7. If the source code under `src` folder changed, you need to stop the notebook docker using `docker stop python_notebook_name` and restart the notebook kernel using `docker run -it --name python_notebook_name --rm -p 8888:8888 -v $(pwd):/python_kernel/notebook opendigger-jupyter-python:1.0` to reload the sorce code.
```
[the remark this answers is not in the document]
Comment (frank-zsy) [reply sample_data/README.md:122]: Actually `--rm` flag is used to remove the container entity after stop the container.

The thing is for Node.js kernel, after source code change, we only need to re-compile the code and restart the kernel in JupyterLab, we don't really need to restart the container to make the change take effect.

[the remark this answers is not in the document]
Comment (zhicheng-ning) [reply sample_data/README.md:122]: I got, many thanks!

**Comment by frank-zsy** [comment]
I think we can merge this PR now and consider how to improve the docs later.

Thanks for the contribution @yoyo-wu98 

/approve

**Comment by yoyo-wu98** [comment]
Many thx:)

**Pull request merged**